README.md

```diff
@@ -298,7 +309,65 @@ OPTIONS
   --help                 show CLI help
 ```
 
-_See code: [src/commands/instance/spawn.ts](https://github.com/kuzzleio/kourou/blob/v0.5.0/src/commands/instance/spawn.ts)_
+_See code: [src/commands/instance/spawn.ts](https://github.com/kuzzleio/kourou/blob/v0.6.0/src/commands/instance/spawn.ts)_
+
+## `kourou vault:add SECRETS-FILE KEY VALUE`
+
+Add an encrypted key to a secrets file
+
+```
+USAGE
+  $ kourou vault:add SECRETS-FILE KEY VALUE
+
+ARGUMENTS
+  SECRETS-FILE  Encrypted secrets file
+  KEY           Path to the key (lodash style)
+  VALUE         Value to encrypt
+
+OPTIONS
+  --vault-key=vault-key  [default: fHaHTUrdbW9fQWmpme1DMAfUCRAw55FRMa3d6jljFXY=] Kuzzle Vault Key (or KUZZLE_VAULT_KEY)
+```
+
+_See code: [src/commands/vault/add.ts](https://github.com/kuzzleio/kourou/blob/v0.6.0/src/commands/vault/add.ts)_
+
+## `kourou vault:encrypt FILE`
+
+Encrypt an entire file
+
+```
+USAGE
+  $ kourou vault:encrypt FILE
+
+ARGUMENTS
+  FILE  File containing unencrypted secrets
+
+OPTIONS
+  -f, --force                    Overwrite the output file if it already exists
+  -o, --output-file=output-file  Output file (default: <file>.enc.json)
+
+  --vault-key=vault-key          [default: fHaHTUrdbW9fQWmpme1DMAfUCRAw55FRMa3d6jljFXY=] Kuzzle Vault Key (or
```

Comment:
Isn't that weird? Shouldn't this argument be required instead?

(same goes for the other one below)

Reply:
Hmm I think I found a bug in ocliff because the default value for this argument is the environment variable `KUZZLE_VAULT_KEY` (see [here](https://github.com/kuzzleio/kourou/pull/5/files#diff-17d6855aad179fe2d683d9224cb6a4c6R14))
I have to fix the README